### ISSUE TITLE
Migrate hearing comment files from 2009 and old FR notice to CMS

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#5767]
### Summary

**What we're after:**
_We are getting off transition and moving some stray files to the CMS with OGC and OCS' blessing._

### Related issues

- #5642 

### Action Items

~Files to migrate and link on https://www.fec.gov/legal-resources/policy-other-guidance/comments-received-notice-public-hearing-agency-procedures-and-policies-2009/:
http://transition-stage.fec.gov/law/policy/enforcement/2009/comments/comm32.pdf
http://transition-stage.fec.gov/law/policy/enforcement/2009/comments/comm3.pdf
http://transition-stage.fec.gov/law/policy/enforcement/2009/comments/comm4.pdf
http://transition-stage.fec.gov/law/policy/enforcement/2009/comments/comm5.pdf~

This file will be migrated to the CMS and linked on https://www.fec.gov/about/reports-about-fec/agency-operations/:
https://transition.fec.gov/pdf/FR66n132p35978.pdf

### Completion criteria

- [x] 508 compliant versions of files listed above uploaded to CMS
~- [ ] 2009 hearing comments page updated with links to those files~ - not needed; see comment
- [x] Document page created for the old FR notice and included in Agency Reports feed
- [x] Redirects merged for files and [redirect spreadsheet updated](https://docs.google.com/spreadsheets/d/1vzN-wZlS8K8ZyfExSV3hwXDenDEN9KyyAUJFTI6OjKU/edit#gid=687896915)
- [x] Testing ticket created for redirects

### Future work

- Testing the redirects
- Removing those files from transition

**Comment by dorothyyeager** [comment]
The hearing files turned out to be included in the large index file linked at the bottom of the page. We will redirect to the file that contains them. No need to migrate those files individually.

**Comment by dorothyyeager** [comment]
Work completed. Once connected PR is merged; this ticket can be closed.
